x/bank/v2/README.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
maybe this is avoided for now to not get registered on the docs site

Reply:
it for main, so I think it's fine tbh